src/components/AboutUs/TimelineSection.js

```diff
@@ -8,254 +8,306 @@ import Pin from '../../images/about-us/timeline/pin.svg';
 import ImgOpt from '../../utils/ImageOptimise.js';
 
 const useStyles = makeStyles((theme) => ({
-    title: {
-        fontFamily: 'Josefin Sans',
-        fontWeight: 500,
-        fontSize: '32px',
-        lineHeight: '57px',
-        padding: '0px 20px',
-        // Tablet Above to Laptop/ Desktop
-        [theme.breakpoints.between('sm', 'md')]: {
-            fontSize: '28px',
-        },
-        // Tablet below to Mobile
-        [theme.breakpoints.down('sm')]: {
-            fontSize: '18px',
-        },
+  titleContainer: {
+    marginTop: '160px',
+    // Tablet Above to Laptop/ Desktop
+    [theme.breakpoints.between('sm', 'md')]: {
+      marginTop: '96px',
     },
-    yearTag: {
-        filter: 'drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25))',
-        position: 'relative',
-        bottom: '275px',
-        marginRight: '20px',
-        backgroundColor: '#873FE2',
-        borderRadius: '300px',
-        padding: '5px 30px',
-        fontSize: '25px',
-        color: 'white',
-        borderStyle: 'none',
-        fontFamily: 'Futura',
-        [theme.breakpoints.down('sm')]: {
-            display: 'none',
-        },
+    [theme.breakpoints.down('sm')]: {
+      marginTop: '69px',
     },
-    yearTagSm: {
-        filter: 'drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25))',
-        padding: '5px 25px',
-        fontSize: '14px',
-        backgroundColor: '#873FE2',
-        borderRadius: '300px',
-        color: 'white',
-        borderStyle: 'none',
-        fontFamily: 'Futura',
-        display: 'none',
-        [theme.breakpoints.down('sm')]: {
-            display: 'block',
-            marginRight: 'calc(100% - 82px)'
-        },
+  },
+  tealBackground: {
+    background: 'linear-gradient(180deg, #F5FDFD 0%, #FFFFFF 50%)',
+    padding: '50px 20px 0px 0px',
+
+  },
+  title: {
+    fontFamily: 'Josefin Sans',
+    fontWeight: 600,
+    fontSize: '32px',
+    lineHeight: '57px',
+    padding: '0px 20px',
+    // Tablet Above to Laptop/ Desktop
+    [theme.breakpoints.between('sm', 'md')]: {
+      fontSize: '28px',
+      lineHeight: '38px',
     },
-    pinStyle: {
-        display: 'inline-block',
-        height: 300,
-        // Tablet Above to Laptop/ Desktop
-        // Tablet below to Mobile
-        [theme.breakpoints.down('sm')]: {
-            height: 200,
-        },
+    // Tablet below to Mobile
+    [theme.breakpoints.down('sm')]: {
+      fontSize: '18px',
+      lineHeight: '29px',
     },
-    eventTitle: {
-        fontFamily: 'Josefin Sans',
-        fontWeight: 600,
-        fontSize: '28px',
-        lineHeight: '44px',
-        textAlign: 'left',
-        // Tablet Above to Laptop/ Desktop
-        [theme.breakpoints.between('sm', 'md')]: {
-            fontSize: '25px',
-        },
-        // Tablet below to Mobile
-        [theme.breakpoints.down('sm')]: {
-            fontSize: '14px',
-        },
+  },
+  yearCol: {
```

Comment:
align with rest of text (shift right)

Reply:
The red parts are deleted, so everything in green should align with each other